pkg/ccl/storageccl/export_storage.go

```diff
@@ -678,9 +745,10 @@ func makeAzureStorage(conf *roachpb.ExportStorage_Azure) (ExportStorage, error)
 	}
 	blobClient := client.GetBlobService()
 	return &azureStorage{
-		conf:      conf,
-		container: blobClient.GetContainerReference(conf.Container),
-		prefix:    conf.Prefix,
+		conf:             conf,
+		container:        blobClient.GetContainerReference(conf.Container),
+		operationTimeout: timeoutSetting.Get(&settings.SV),
```

Comment:
We generally don't want to store the value of reading a setting, but rather wait to read it when we use it, e.g. below in WriteFile.

Reply:
These storages get recreated every time import/backup/restore executes, so I think it is ok that they take a snapshot of the settings as they appeared at the time the storage was created. Also if we move this to the http.Client, doing this kind of just-in-time setting will be annoying.